src/test/API.test.js

```diff
@@ -45,7 +45,8 @@ describe('User API methods', () => {
       .expect(201);
 
     // eslint-disable-next-line no-underscore-dangle
```

Comment:
I think this can be deleted since there is no underscore in the next line anymore